.ci/cico_build.sh

```diff
@@ -52,6 +52,9 @@ source ${ABSOLUTE_PATH}/../config
 
 runBuild "cd ${ABSOLUTE_PATH} && bash ./cico_do_build_che.sh $*"
 if [ $? -eq 0 ]; then
+  if [ "$TARGET" == "rh-aut" ]; then
```

Comment:
It is still not clear for me what exactly is `rh-aut` target and why the name is so weird cc: @riuvshin

Reply:
I had to somehow separate this from the normal rhel build, because I'm assigning different variables and I wanted it to be visible in the `ci.centos.org` jenkins file, that the job is meant for automated testing.

Reply:
ok, got it `rh-aut`  =`rh-automation` . Frankly I would prefer better naming like `rh-integration-test`

Reply:
Could do that, though I would like to rather keep it short.

Reply:
it it short, but not at all clear (at very least for me)

Reply:
renamed